### PR TITLE
Code cleanup D3D

### PR DESCRIPTION
**Pull request by Loggan08**
+ use of c++ style casts

### CODE REVIEW

**Comment by Tilka** [comment]
Please don't replace short types with `auto`.

**Comment by Loggan08** [comment]
Okay, will undo

File: Source/Core/VideoBackends/D3D/D3DBase.cpp [hunk 1]
```diff
@@ -427,25 +427,25 @@ void Close()
...
 	else /*if(featlevel == D3D_FEATURE_LEVEL_10_0)*/ return "vs_4_0";
 }
 
-const char* GeometryShaderVersionString()
-{
-	if (featlevel == D3D_FEATURE_LEVEL_11_0) return "gs_5_0";
-	else if (featlevel == D3D_FEATURE_LEVEL_10_1) return "gs_4_1";
-	else /*if(featlevel == D3D_FEATURE_LEVEL_10_0)*/ return "gs_4_0";
-}
+	const char* GeometryShaderVersionString()
```
Comment (phire) [inline Source/Core/VideoBackends/D3D/D3DBase.cpp]: Please keep the indentation consistent.

File: Source/Core/VideoBackends/D3D/D3DBase.cpp [hunk 2]
```diff
@@ -427,25 +427,25 @@ void Close()
...
+		else /*if(featlevel == D3D_FEATURE_LEVEL_10_0)*/ return "gs_4_0";
+	}
 
-const char* PixelShaderVersionString()
-{
-	if (featlevel == D3D_FEATURE_LEVEL_11_0) return "ps_5_0";
-	else if (featlevel == D3D_FEATURE_LEVEL_10_1) return "ps_4_1";
-	else /*if(featlevel == D3D_FEATURE_LEVEL_10_0)*/ return "ps_4_0";
-}
+	const char* PixelShaderVersionString()
```
Comment (phire) [inline Source/Core/VideoBackends/D3D/D3DBase.cpp]: Also here.

File: Source/Core/VideoBackends/D3D/D3DBase.cpp (hunk 1 above)
Comment (Loggan08) [inline Source/Core/VideoBackends/D3D/D3DBase.cpp]: Got it

**Comment by Loggan08** [comment]
Alright I'm back home, will spend the entire day doing as many (safe) tweaks i am able to do

**Pull request closed without merging**